packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.styles.ts

```diff
@@ -1,8 +1,12 @@
 import { IGroupHeaderStyleProps, IGroupHeaderStyles } from './GroupHeader.types';
 import { getGlobalClassNames, getFocusStyle, FontSizes, IStyle, AnimationVariables, FontWeights, IconFontSizes } from '../../Styling';
+import { DEFAULT_ROW_HEIGHTS, DEFAULT_CELL_STYLE_PROPS } from '../DetailsList/DetailsRow.styles';
+// For every group level there is a GroupSpacer added. Importing this const to have the source value in one place.
+import { SPACER_WIDTH as EXPAND_BUTTON_WIDTH } from './GroupSpacer';
 
 const GlobalClassNames = {
   root: 'ms-GroupHeader',
+  compact: 'ms-GroupHeader--Compact',
```

Comment:
super nit, but for consistency within these class names I'd prefer `--compact`

Reply:
😄 I tried being consistent here with DetailsList https://github.com/OfficeDev/office-ui-fabric-react/blob/940979dd0ed4c31213850fcca28c6246a4fcc350/packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.styles.ts#L6
But I can change it as it's a new addition, so not breaking anyone))

Reply:
Ha. I should have known better given your attention to detail.

This is an interesting one. On one hand I'd like to clean it up now, but it _could_ be considered breaking in the `DetailsList` case for `6.x`. Perhaps we can commit against a `7.0` branch and communicate it during release notes. Then they'd be in-sync after some time.

Reply:
Left a comment for TODO in 7.0!